apps/appsets/openstack.yaml

```diff
@@ -38,7 +38,7 @@ spec:
                   repoURL: https://tarballs.opendev.org/openstack/openstack-helm
                   openstackRelease: 2024.2
                   # renovate: datasource=custom.openstackhelm depName=placement
-                  chartVersion: 0.3.17
+                  chartVersion: 2024.2.0
```

Comment:
This is Vlad's rolling release version. You've got to pick one with the git suffix like I did for keystone above for it to remain stable.